remote-wallet/src/ledger_error.rs

```diff
@@ -0,0 +1,86 @@
+use num_derive::FromPrimitive;
+use thiserror::Error;
+
+#[derive(Error, Debug, Clone, FromPrimitive, PartialEq)]
+pub enum LedgerError {
+    #[error("Solana app not open on Ledger device")]
+    NoAppResponse = 0x6700,
+
+    #[error("Ledger sdk exception")]
+    SdkException = 0x6801,
+
+    #[error("Ledger invalid parameter")]
+    SdkInvalidParameter = 0x6802,
+
+    #[error("Ledger overflow")]
+    SdkExceptionOverflow = 0x6803,
+
+    #[error("Ledger security exception")]
+    SdkExceptionSecurity = 0x6804,
+
+    #[error("Ledger invalid crc")]
```

Comment:
@t-nelson , I'm not clear on what this one is, as I didn't find it implemented in our app code. Can you help me with a better human-readable string?